specifications/api/swagger.yaml

```diff
@@ -2140,32 +2159,30 @@ paths:
       description: |
         <p align="right">status: <strong>stable</strong></p>
 
-        Calculate the exact cost of migrating from a Byron wallet to a new
-        wallet.
+        Calculate the exact cost of migrating from a Byron wallet to
+        specified addresses.
       parameters:
-        - *parametersWalletId
-      responses: *responsesGetByronWalletMigrationInfo
+        - <<: *parametersWalletId
+          name: walletId
+      responses: *responsesGetWalletMigrationInfo
 
-  /byron-wallets/{sourceWalletId}/migrations/{targetWalletId}:
     post:
       operationId: migrateByronWallet
       tags: ["Byron Migrations"]
       summary: Migrate
       description: |
         <p align="right">status: <strong>stable</strong></p>
 
-        Migrate all funds from a Byron wallet to a new-style wallet.
+        Migrate all funds from a Byron wallet to specified addresses.
```

Comment:
```suggestion
        Migrate all funds from a Byron wallet to a set of addresses.
```

Reply:
corrected in https://github.com/input-output-hk/cardano-wallet/pull/1664/commits/cf2b9e58c71fd7e80f8a9e1b9073348dfba3a67e